### DISCUSSION TITLE
PLY import; what is Mask::IOM_EDGEINDEX?

### DISCUSSION THREAD

**Discussion by 12AT7** [cnr-isti-vclab/meshlab#1018] [Q&A] [answered]
I could use a hint about how to get this Ply ("edge.ply") to import into Meshlab:
```
ply
format ascii 1.0
element vertex 4
property float x
property float y
property float z
element edge 4
property uint vertex1
property uint vertex2
end_header
0.0 0.0 0.0
1.0 0.0 0.0
1.0 1.0 0.0
0.0 1.0 0.0
0 1
1 2
2 3
3 0
```
Running `meshlab edge.ply` fails with the message
```
meshlab: ./vcglib/wrap/io_trimesh/import_ply.h:736: static int vcg::tri::io::ImporterPLY<OpenMeshType>::Open(OpenMeshType&, const char*, vcg::tri::io::PlyInfo&) [with OpenMeshType = CMeshO]: Assertion `pi.mask & Mask::IOM_EDGEINDEX' failed.
```
I am eager to change my PLY outputs to be compatible with Meshlab, but I cannot decipher this error message.  It sounds like I need some extra metadata in there?

This is my version: `MeshLab_64bit_fp v2020.03+dfsg1` which is from the stock Ubuntu 20.04 repository using `apt`.

Thanks for any help if someone can spot what my bug is.

**Comment by alemuntoni** [comment, marked as answer]
This is a bug in the ply importer that has been already solved in previous versions of meshlab (2020.03 is quite old now).
You can download the last version in the release tab of this repository.

**Reply by 12AT7** [reply]
Ah, thank you!  This is working for me now.
